MonoGame.Framework.Content.Pipeline/ContentProcessorContext.cs

```diff
@@ -24,7 +24,7 @@ public abstract class ContentProcessorContext
         /// <summary>
         /// Gets the logger interface used for status messages or warnings.
         /// </summary>
-        public abstract ContentBuildLogger Logger { get; }

```

Comment:
We can't change this to `IContentBuildLogger` because it's part of the public API. Old processors throw `System.MissingMethodException`.